critiquebrainz/ws/oauth/provider.py

```diff
@@ -106,15 +112,14 @@ def persist_grant(client_id, code, scopes, expires, redirect_uri, user_id):
 
     @staticmethod
     def persist_token(client_id, scope, refresh_token, access_token, expires, user_id):
-        token = OAuthToken(client_id=client_id,
-                           scopes=scope,
-                           access_token=access_token,
-                           refresh_token=refresh_token,
-                           expires=expires,
-                           user_id=user_id)
-
-        db.session.add(token)
-        db.session.commit()
+        token = db_oauth_token.create(
+            client_id=client_id,
+            scopes=scope,
+            access_token=access_token,
+            refresh_token=refresh_token,
+            expires=expires,
+            user_id=user_id,
+        )
         return token
```

Comment:
Here you can just do `return db_oauth_token.create(...)` without creating a variable.